src/gui/mzroll/projectdockwidget.cpp

```diff
@@ -659,7 +666,7 @@ void ProjectDockWidget::saveSQLiteProject(QString filename)
 
 void ProjectDockWidget::saveSQLiteProject()
 {
-    if (_mainwindow->fileLoader->sqliteProjectIsOpen()) {
+    if (!_currentProjectIsTemporary()) {
```

Comment:
if `sqliteProjectisOpen` is false, then it'll try to `getLastOpenedProject()`. Is that what you intended?